pages/docs/reference/basic-types.md

```diff
@@ -324,6 +326,25 @@ x[0] = x[1] + x[2]
 
 </div>
 
+<div class="sample" markdown="1" theme="idea" data-highlight-only>
+
+```kotlin
+// Array of int of a size of N
+val arr = IntArray(N)
+
+// e.g. initialise the values in the array with a constant
+// Array of int of a size of N with values initialised to 42
+val arr = IntArray(N, 42)
```

Comment:
This is not valid Kotlin code. To initialize all elements with a constant, you need to use `IntArray(N) { 42 }).`

Reply:
Yes, fixed with another commit.  Thanks (yes, very sloppy of me - sorry about that - checked and formatted in IDE this time)